api/services/elastigroup/azure/spotVms/paths/groupVmDeployments.yaml

```diff
@@ -25,7 +25,7 @@ post:
       description: "Bad Request"
 
 get:
-  summary: "List Deployments"
+  summary: "Get All Deployments"
```

Comment:
already fixed by the previous PR.
please align this branch with the most up-to-date master branch